R/bm-R-to-arrow.R

```diff
@@ -0,0 +1,37 @@
+#' Benchmark file writing
+#'
+#' @section Parameters:
+#' * `source` A known-file id, or a CSV(?) file path to read in
+#' * `format` One of `c("parquet", "feather", "fst")`
+#' * `compression` One of `c("uncompressed", "snappy", "zstd")`
+#' * `input` One of `c("arrow_table", "data_frame")`
+#'
+#' @export
+df_to_table <- Benchmark("df_to_table",
+  setup = function(source = names(known_sources), ...) {
```

Comment:
remove ... now?

Reply:
Yes, thanks for catching that. I had removed all of them on the _other_ branch but not here